NOTES.md

```diff
@@ -0,0 +1,43 @@
+| Command | Description |
```

Comment:
This file shouldn't be in here?  It's my notes that were committed in error during the Cobra PR and was subsequently removed.